test/e2e/framework/util.go

```diff
@@ -71,9 +71,23 @@ import (
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 )
 
+const (
+	// Minimal number of nodes for the cluster to be considered large.
+	largeClusterThreshold = 100
+
+	// TODO(justinsb): Avoid hardcoding this.
+	awsMasterIP = "172.20.0.9"
+
+	// AllContainers specifies that all containers be visited
+	// Copied from pkg/api/v1/pod to avoid pulling extra dependencies
+	AllContainers = InitContainers | Containers | EphemeralContainers
+)
+
+// DEPRECATED constants. Use the timeouts in framework.Framework instead.
```

Comment:
Ideally we'd remove these constants from here in favor of the values in framework.Framework.Timeouts. However, these consts are used in **too many places** throughout the test suite and changing all at once would make this PR too big (I tried that).

So the approach I found more convenient was to deprecate the consts in `util.go` and and remove them over time, once there isn't any test using them any more.